### DISCUSSION TITLE
/cmi staffmode .... inventory switch

### DISCUSSION THREAD

**Discussion by CreeperHero** [Zrips/CMI#7709] [Feature Requests]
A command that switches the inventory and effects of the user 

normal <---> staff

This way staff members won't need to empty inventory, amour, or reset effects when stopping playing to help someone need.

**Comment by mrfloris** [comment]
I think you might be able to make a `/cmi aliaseditor `command for staffmode, that uses `/cmi saveditems` to  achieve this?
```yaml
> cmi checkcommand saveditems
/cmi saveditems [save/get/remove/list] (savedItemName) (-t:playerName) (-a:amount) (-c:category/all) (-s)
>
```

**Reply by CreeperHero** [reply]
That seems to only save one item. I want to save the entire inventory, amour, and status effects, then clear them and swap to one called staff or something. After your done with your staff things (maybe an hour or two days), I want to reverse it putting the inventory, amour, and status effects back the way they was.

**Comment by mrfloris** [comment]
Oh yeah, fair point. Maybe invsave, invload can then do the trick, sorry.
Ahk, yes, you don't just want items, you want to have previous exp and modes and all sorts. okay. 

Without existing features, i'd try to use the placeholders to store temporary usermeta data and have a /staffmode custom alias that runs this, detecting if you /staffmode on|off, I wonder how much can be supported. 

I like the suggestion though.
